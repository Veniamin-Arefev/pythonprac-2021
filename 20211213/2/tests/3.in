(rand := __import__("random"), rand.seed("Taki i eche odin testik"), [rand.randint(-100, 100) for i in range(63)])[2]