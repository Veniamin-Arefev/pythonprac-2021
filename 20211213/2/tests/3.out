[-91, -88, -87, -86, -85, -83, -82, -79, -76, -74, -72, -62, -60, -58, -56, -56, -54, -53, -52, -46, -41, -36, -30, -29, -25, -16, -16, -6, -3, 10, 10, 11, 12, 17, 20, 21, 23, 26, 27, 29, 29, 31, 35, 36, 37, 40, 44, 46, 53, 54, 54, 58, 58, 58, 63, 69, 74, 84, 84, 86, 87, 91, 92]
